README.md

```diff
@@ -17,6 +17,10 @@ Ansible role for a bigbluebutton installation (following the documentation on ht
 | `bbb_api_demos_enable` | enable installation of the api demos | `no` |
 | `bbb_nodejs_version` | version of nodejs to be installed | `8.x` |
 | `bbb_system_locale` | the system locale to use | `en_US.UTF-8` |
+| `bbb_greenlight_reg` | Registration mode | `open` | Greenlight registration mode: open, invite, aprroval |
+| `bbb_cam_rec` | Disable recording | `false` | Disable recording by default: false/true |
```

Comment:
I would also suggest to avoid the double-negative form:

```suggestion
| `bbb_enable_rec` | Enable recording | `true` | Enable recording by default: false/true |
```